.github/workflows/check-spelling.yml

```diff
@@ -23,3 +23,32 @@ jobs:
 
       - name: Run cargo-spellcheck
         run: cargo spellcheck --cfg=./spellcheck/era.cfg --code 1
```

Comment:
Are you sure we need both? 
could we just run `zk spellcheck`?

Reply:
Only reason I separated them was the I kept getting a failure when installing `cargo-spellcheck` on the matterlabs-ci-runner. So I separated so spell-check was installed and ran on ubuntu-latest.